### ISSUE TITLE
Ensure `send_exception` and `set_exception` take the same arguments

### ISSUE THREAD

**Issue by matsimitsu** [appsignal/appsignal-ruby#281]
It's weird that one accepts tags/namespace and the other doesnt, we should make them both accept the same arguments, so you can easily switch them out. 

Right now:

```ruby
def send_error(error, tags = nil, namespace = Appsignal::Transaction::HTTP_REQUEST)
[...]
def set_error(exception)
```

**Comment by tombruijn** [comment]
Could we maybe also change that the behavior of the two and combined them?

If there's no current transaction, a new one is created, the event is added and the transaction is completed.
If a transaction is active the error is added to that transaction.

That way we eliminate the confusion between the two.

**Comment by tombruijn** [comment]
Oh.. Just realized we already had an issue for that: https://github.com/appsignal/appsignal-ruby/issues/176

**Comment by tombruijn** [comment]
Fixed in #317
